linux_os/guide/services/ssh/ssh_server/sshd_use_directory_configuration/bash/shared.sh

```diff
@@ -1,12 +1,15 @@
 # platform = multi_platform_all
 
 {{% set target_file = "/etc/ssh/sshd_config.d/sshd_config_original.conf" -%}}
+{{% set base_config = "/etc/ssh/sshd_config" -%}}
 if test -f {{{ target_file}}}; then
 	{{{ die("Remediation probably already happened, '" ~ target_file ~ "' already exists, not doing anything.", action="false") }}}
+elif grep -Eq '^\s*Include\s+/etc/ssh/sshd_config\.d/\*\.conf' {{{ base_config }}} && ! grep -Eq '^\s*Match\s' {{{ base_config }}}; then
+	{{{ die("Remediation probably already happened, {{{ base_config }}} already contains the include directive.", action="false") }}}
```

Comment:
This line needs some more Jinja foo, the remediation is printing:
`Remediation probably already happened, {{{ base_config }}} already contains the include directive.`

Reply:
Good catch!